openmetadata-ui/src/main/resources/ui/src/pages/Persona/PersonaDetailsPage/PersonaDetailsPage.tsx

```diff
@@ -277,7 +278,11 @@ export const PersonaDetailsPage = () => {
                 ? t('message.remove-default-persona-description')
                 : t('message.set-default-persona-menu-description')
             }
-            icon={(isDefault ? XOutlined : CheckCircleOutlined) as SvgComponent}
+            icon={
+              (isDefault
+                ? CloseCircleOutlined
+                : CheckCircleOutlined) as SvgComponent
```

Comment:
we are also using CheckCircleOutlined from the Antd icons, let's use the svg icons from the project itself!